services/spar/src/Spar/Scim/User.hs

```diff
@@ -664,9 +684,13 @@ scimFindUserByHandle mIdpConfig stiTeam hndl = do
 -- successful authentication with their SAML credentials.
 scimFindUserByEmail :: Maybe IdP -> TeamId -> Text -> MaybeT (Scim.ScimHandler Spar) (Scim.StoredUser ST.SparTag)
 scimFindUserByEmail mIdpConfig stiTeam email = do
+  lift $ lift $ Log.debug (Log.msg $ "scimFindUserByEmail 1: " <> (show (stiTeam, email)))
   veid <- mkValidExternalId mIdpConfig (pure email)
+  lift $ lift $ Log.debug (Log.msg $ "scimFindUserByEmail 2: " <> (show veid))
   uid <- MaybeT . lift $ ST.runValidExternalId withUref withEmailOnly veid
-  brigUser <- MaybeT . lift . Brig.getBrigUserAccount $ uid
+  lift $ lift $ Log.debug (Log.msg $ "scimFindUserByEmail 3: " <> (show uid))
+  brigUser <- MaybeT . lift . Brig.getBrigUserAccount Brig.WithPendingInvitations $ uid
+  lift $ lift $ Log.debug (Log.msg $ "scimFindUserByEmail 4: " <> (show brigUser))
```

Comment:
is this all too much debug information?  (i'm not sure)

Reply:
Yes i think so, I removed it.